db/migrate/20230117002110_create_sidebar_url.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class CreateSidebarUrl < ActiveRecord::Migration[7.0]
+  def change
+    create_table :sidebar_urls do |t|
+      t.string :name, null: false
+      t.string :value, null: false
+      t.timestamps
+    end
```

Comment:
I wonder if we should be using a generic primary key here and instead set the primary key to point towards the `sidebar_section_link` record. In other words, a row in the `sidebar_urls` table can only exist when it references a valid `sidebar_section_link` record.

Reply:
Discussed this with @lis2 and realised this is not practical/useful.